app/views/layouts/base.html.erb

```diff
@@ -57,13 +57,14 @@
   <div class="usa-overlay"></div>
   <%= yield(:mobile_nav) if content_for?(:mobile_nav) %>
   <%= render 'shared/banner' %>
-  <main class="site-wrap bg-primary-lighter" id="main-content">
-    <div class="grid-container padding-y-4 tablet:padding-y-8 <%= local_assigns[:disable_card].present? ? '' : 'card' %>">
-      <%= yield(:pre_flash_content) if content_for?(:pre_flash_content) %>
-      <%= render FlashComponent.new(flash: flash) %>
-      <%= content_for?(:content) ? yield(:content) : yield %>
-    </div>
-  </main>
+
+  <%= content_tag(local_assigns[:custom_main].present? ? 'div' : 'main', { class: 'site-wrap bg-primary-lighter', id: local_assigns[:custom_main].present? ? 'main-wrapper' : 'main-content' }) do %>
+      <div class="grid-container padding-y-4 tablet:padding-y-8 <%= local_assigns[:disable_card].present? ? '' : 'card' %>">
+        <%= yield(:pre_flash_content) if content_for?(:pre_flash_content) %>
+        <%= render FlashComponent.new(flash: flash) %>
+        <%= content_for?(:content) ? yield(:content) : yield %>
+      </div>
+  <% end %>
```

Comment:
looks like this has an indent by 4 instead of by 2? also maybe let's put the `content_tag` stuff on separate lines so it's easier to follow?
```suggestion
  <%= content_tag(
        local_assigns[:custom_main].present? ? 'div' : 'main',
        class: 'site-wrap bg-primary-lighter',
        id: local_assigns[:custom_main].present? ? 'main-wrapper' : 'main-content'
      ) do %>
    <div class="grid-container padding-y-4 tablet:padding-y-8 <%= local_assigns[:disable_card].present? ? '' : 'card' %>">
      <%= yield(:pre_flash_content) if content_for?(:pre_flash_content) %>
      <%= render FlashComponent.new(flash: flash) %>
      <%= content_for?(:content) ? yield(:content) : yield %>
    </div>
  <% end %>
```